include/boost_sml/sml_transition_graph.h

```diff
@@ -27,7 +27,7 @@ class SmlTransitionGraph: public graph_t
 {
 public:
   using edge_t = std::pair<std::string, std::string>;
-  static constexpr vertex_descriptor NIL = std::numeric_limits<vertex_descriptor>::max();
+  static const vertex_descriptor NIL = std::numeric_limits<vertex_descriptor>::max();
```

Comment:
This will still fail, before c++17 we have to redefine it outside of class

```cpp
constexpr SmlTransitionGraph::vertex_descriptor SmlTransitionGraph::NIL;
} // namespace sml_transition_graph
```

Reply:
Tried to make this just const. Grr.

Reply:
Sorry, I forgot to add it, I've no idea why my machine's compiler wasn't failing, I did test it in a docker container and managed to fix it with the above changes